### DISCUSSION TITLE
How to iterate through all the items of I.grabTextFromAll

### DISCUSSION THREAD

**Discussion by RajeshVMVolvo** [codeceptjs/CodeceptJS#2910] [General]
For eg. 
var vehicle = await I.grabTextFromAll(locator);
I can grab text of all elements , now I need to check whether 'car' exits or not. How can i do it ..pls suggest

**Comment by kobenguyent** [comment]
oh @RajeshVMVolvo the returned value `vehicle` should be an array and you could basically do a loop through that array and assert what you need.

**Reply by RajeshVMVolvo** [reply]
yes i copied to another array & iterated.
var arrVehicle = [] ; 
Array.prototype.push.apply(arrVehicle ,vehicle);
Then used array methods to itearate
